### DISCUSSION TITLE
td-agent throwing error in logs

### DISCUSSION THREAD

**Discussion by Samrudh-advani** [fluent/fluentd#3821] [Q&A]
I have installed td-agent, and when checking the logs i can see the below error 

"2022-07-15 10:33:57 -0700 [info]: #0 'flush_interval' is configured at out side of <buffer>. 'flush_mode' is set to 'interval' to keep existing behaviour
2022-07-15 10:33:57 -0700 [error]: #0 config error file="/etc/td-agent/td-agent.conf" error_class=Fluent::ConfigError error="Using Elasticsearch client 8.1.2 is not compatible for your Elasticsearch server. Please check your using elasticsearch gem version and Elasticsearch server."
2022-07-15 10:33:57 -0700 [error]: Worker 0 finished unexpectedly with status 2
2022-07-15 10:33:57 -0700 [info]: Received graceful stop"

the logs keep repeating itself. 

this instance has version td-agent 4.3.1 fluentd 1.14.6

my other instance is working fine with same conf file and it is running with version td-agent 1.11.5

i want to revert back the changes of td-agent and remove fluentd. 

can someone help me with the same

**Comment by cosmo0920** [comment]
> 2022-07-15 10:33:57 -0700 [error]: #0 config error file="/etc/td-agent/td-agent.conf" error_class=Fluent::ConfigError error="Using Elasticsearch client 8.1.2 is not compatible for your Elasticsearch server. Please check your using elasticsearch gem version and Elasticsearch server."

This line says that your using ES plugin's dependent elasticsearch gem and your ES cluster versions are not matched.
The latest td-agent uses elasticsearch 8.1.2 gem but this gem does not compatible Elasticsearch 7 or earlier.
You have to install the corresponding version of elasticsearch gem for your ES cluster.

**Comment by Samrudh-advani** [comment]
can you pls help me with the document for installation of ES cluster with the latest version


Thanks

**Reply by Samrudh-advani** [reply]
hi,

I'm facing issues while installing td-agent. below is the command I'm executing -

curl -L https://toolbelt.treasuredata.com/sh/install-redhat-td-agent3.sh | sh

facing errors with the package dependency. attaching screenshot of the error. can someone help me with the same


using centos7 and not finding right packages to install. 

pls guide me through this - 

<img width="422" alt="fluentd" src="https://user-images.githubusercontent.com/79824854/183928156-f1923f26-e59e-49bd-9ece-f06c8bc8797e.PNG">


Thanks
